packages/grid/test/column-reordering.test.js

```diff
@@ -378,8 +400,8 @@ describe('reordering grid with columns groups', () => {
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid style="width: 400px; height: 200px;" size="1" column-reordering-allowed>
-        ${[1, 2].map((colgroup) => {
+      <vaadin-grid style="width: 800px; height: 200px;" size="1" column-reordering-allowed>
```

Comment:
Increased the width just to see the whole grid while working on the test case (doesn't affect any tests)